src/Activation/ProductAnalyticsConsentForm.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { FunctionComponent } from "react"
+import {
+  Linking,
+  View,
+  StyleSheet,
+  ScrollView,
+  TouchableOpacity,
+} from "react-native"
+import { useNavigation } from "@react-navigation/native"
+import { useTranslation } from "react-i18next"
+
+import { useConfigurationContext } from "../ConfigurationContext"
+import { useProductAnalyticsContext } from "../ProductAnalytics/Context"
+import { ActivationStackScreens, useStatusBarEffect } from "../navigation"
+
+import { Text } from "../components"
+import { Colors, Typography, Spacing, Buttons } from "../styles"
+
+const ProductAnalyticsConsentForm: FunctionComponent = () => {
+  useStatusBarEffect("dark-content", Colors.secondary.shade10)
+  const { t } = useTranslation()
+  const navigation = useNavigation()
+  const {
+    healthAuthorityName,
+    healthAuthorityPrivacyPolicyUrl,
+  } = useConfigurationContext()
+  const { updateUserConsent } = useProductAnalyticsContext()
+
+  const handleOnPressButton = async () => {
+    updateUserConsent(true)
+    navigation.navigate(ActivationStackScreens.ActivationSummary)
+  }
+
+  const handleOnPressMaybeLater = () => {
+    navigation.navigate(ActivationStackScreens.ActivationSummary)
+  }
+
+  const handleOnPressPrivacyPolicy = () => {
+    Linking.openURL(healthAuthorityPrivacyPolicyUrl)
+  }
+  const privacyPolicyLinkText = t("product_analytics.privacy_policy")
+
+  const buttonText = t("product_analytics.i_understand_and_consent")
```

Comment:
Is there a reason these bits of text are pulled out?

Reply:
artifact of the refactor, can inline.